docker/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.5"
```

Comment:
Why not 3.7 which is latest?

Reply:
Because there are no need for 3.7. In fact, maybe up to 3.5 is not necessary. Forcing developers to have a specific docker-compose version because of a definition file that does not use any feature, I do not think it's good practice.

Reply:
I tend to use latest available version to avoid being obsoleted by upstream developers. But your reasoning sounds ok to me.